sql/2019/04_Media/04_01a.sql

```diff
@@ -0,0 +1,105 @@
+#standardSQL
+/*
+04_01a
+
+1.22 TB query
+*Lighthouse Lazyloading, resp images, image quality, webp image score score in 1000 percentile points	
+
+this query takes lighthouse data on image performance: 
+    lazyloading (offscreen), 
+    responsiveness, 
+    quality (optimized),
+    format (webp),
+    animated GIF
+    
+    I break down this data into: 
+         scores (from 0-1) 
+         wasted ms in load time
+         wasted KB
+         count of offending items
+    and break them into percentiles - 26 is the median, but 50 seemed 
+    like a good number to graph the data with
+*/
+
+select 
+      APPROX_QUANTILES( offscreenScore ,1000) offscreenscore,
+      APPROX_QUANTILES( repimagesScore ,1000) repimagesScore,
+      APPROX_QUANTILES( optimagesScore ,1000) optimagesScore,
+      APPROX_QUANTILES( webpimagesScore ,1000) webpimagesScore
+from(
+select
```

Comment:
These inner selects do A LOT of unnecessary work that goes unused in the outer select (especially the JSON_EXTRACTs). This query takes over 6 minutes to execute. For query efficiency let's trim this down to only the values used in the outer select (offscreen, repimages, optimages, webpimages).

This comment also applies to the rest of the 01 queries.

Reply:
fixed these.